android/src/main/java/cloud/pace/plugins/cloudsdk/CloudSDK.kt

```diff
@@ -148,6 +150,22 @@ class CloudSDK : Plugin(), AppCallback {
         }
     }
 
+    private fun parseOpeningHours(poiOpeningHours: List<OpeningHours>?): List<OpeningHour> {
+        val result = mutableListOf<OpeningHour>()
+
+        poiOpeningHours?.forEach {
+            if (it.rule == OpeningRule.OPEN) {
+                it.hours.firstOrNull()?.let { hour ->
```

Comment:
Why do we iterate through the hours first and not the days? Wouldn't the result be wrong in this case, i.e. we will have multiple day entries for every opening hour?

Reply:
I did checked results and they were fine for the stations i checked. But you're right, if a day had more than one "Open timespan" we'd have multiple day entries here. Changed it 👍